control-plane/subcommand/server-acl-init/command.go

```diff
@@ -355,19 +342,8 @@ func (c *Command) Run(args []string) int {
 		}
 	}
 
-	// For all of the next operations we'll need a Consul client.
-	serverAddr := fmt.Sprintf("%s:%d", serverAddresses[0], c.flagServerPort)
-	consulClient, err := consul.NewClient(&api.Config{
-		Address: serverAddr,
-		Scheme:  scheme,
-		Token:   bootstrapToken,
-		TLSConfig: api.TLSConfig{
-			Address: c.flagConsulTLSServerName,
-			CAFile:  c.flagConsulCACert,
-		},
-	})
+	consulClient, err := common.GetConsulClientForServer(serverAddresses, c.flagServerPort, scheme, bootstrapToken, c.flagConsulTLSServerName, c.flagConsulCACert, c.log)
```

Comment:
looking at it now, this function was probably not needed since it basically just constructs this struct but it's fine too

Reply:
yeah.. i was middle of the refactor when I kinda realized that it was probably overkill. Ill revert it.